src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -255,6 +263,25 @@ describe('Test suite for dictionary concepts components', () => {
     expect(mapStateToProps(initialState).newName).toEqual(['1']);
     expect(mapStateToProps(initialState).existingConcept).toEqual(existingConcept);
   });
+
+  it('should show the modal when cancel is clicked with unsaved changes', () => {
+    const event = {
+      target: {
+        name: 'datatype',
+        value: 'Complex',
+      },
+    };
+    editConceptWrapper.find('select#datatype').simulate('change', event);
+    editConceptWrapper.find('button#remove').simulate('click');
```

Comment:
Are you clicking the cancel or remove button?

Reply:
I have changed the line to `editConceptWrapper.find('.cancelButton').simulate('click');`